python/pyiceberg/table/sorting.py

```diff
@@ -83,6 +89,9 @@ def set_null_order(cls, values: Dict[str, Any]) -> Dict[str, Any]:
     direction: SortDirection = Field()
     null_order: NullOrder = Field(alias="null-order")
 
+    def __str__(self):
+        return f"{self.transform}({self.source_id}) {self.direction} {self.null_order}"
```

Comment:
Missing `)`.

Reply:
Also, if `transform` is the identity transform, it should be omitted. Rather than `identity(category) ASC NULLS_FIRST` it should be `category ASC NULLS_FIRST`.